test/smoke_test/bcda_client.go

```diff
@@ -145,6 +146,27 @@ func isValidNDJSONFile(filename string) bool {
 	return isValid
 }
 
+func isValidNDJSONText(data string) bool {
```

Comment:
is this function necessary?  can't we just reuse `isValidNDJSONFile`?

Reply:
no, because the output isn't a file.